src/pybind/mgr/tox.ini

```
@@ -84,7 +87,17 @@ commands = {[testenv]commands}
 basepython = python3
 deps =
     autopep8
+modules =
+    cephadm
+    crash
+    devicehealth
+    insights
+    iostat
+    orchestrator
+    prometheus
+    status
+    telemetry
```

Comparing this to the `mypy` env I find the following missing:
```suggestion
modules =
    alerts
    cephadm
    crash
    dashboard
    devicehealth
    insights
    hello
    iostat
    mds_autoscaler
    mgr_module
    mgr_util
    mirroring
    orchestrator
    progress
    prometheus
    rook
    status
    telemetry
    volumes
    zabbix
```

---

the PEP8-clean list is not in sync with that of mypy-clean list yet. they will.